showcase/tests/integration/components/hds/dropdown/list-item/interactive-test.js

```diff
@@ -95,7 +95,7 @@ module(
 
     // CONTENT
 
-    test('it should render the text passed as @text prop', async function (assert) {
+    test('it should not render the text passed as @text prop', async function (assert) {
```

Comment:
question: aren't we aiming for a backward-compatible approach? In this case the `@text` should continue to work (even if deprecated), right?

Reply:
Yes, I think this was a mistake in the name of the test. The test is actually checking the opposite.

Reply:
I think the name of the test was correct, _it should render the text passed as @text prop_, even if it's deprecated, we still support this case.

Reply:
Yes, I meant that I made the mistake and need to revert it. Sorry, that was unclear.